lighthouse-core/report/html/report-styles.css

```diff
@@ -388,10 +388,15 @@
 }
 
 /* Node */
-.lh-node {
-  display: block;
+.lh-node__snippet {
   font-family: var(--monospace-font-family);
   color: hsl(174, 100%, 27%);
+  font-size: 12px;
+}
+.lh-node__title {
+  color: black;
```

Comment:
click the icon in the top left to toggle dark mode. use `--body-text-color` instead